uniemoji.py

```diff
@@ -265,7 +265,9 @@ def _filter(self, query, limit=100):
         for candidate, candidate_info in candidates.items():
             if len(query) > len(candidate): continue
 
-            if query == candidate:
+            candidate_lowercase = candidate.lower()
```

Comment:
Is there a reason to keep the original `candidate` around?
If not, `candidate = candidate.lower()` is simpler

Reply:
I was going to say that it isn't needed except that you'd see `🇸🇪 :flag_se: flag of sweden` (country name starts with lowercase) instead of `🇸🇪 :flag_se: flag of Sweden`, but seems like it actually breaks (emoji is `None` instead of the flag)